src/harness/reference_models/pre_iap_filtering/fss_purge_test.py

```diff
@@ -0,0 +1,73 @@
+#    Copyright 2017 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import json
+import os
+import unittest
+import full_activity_dump
+import fss_purge
```

Comment:
Also here, please put absolute import from the harness/ directory (which is the root python path):
```
from reference_models.pre_iap_filtering import fss_purge
```

Reply:
Modified   to absolute path